core/forms.py

```diff
@@ -143,3 +143,11 @@ class Meta:
         fields = "__all__"
 
     updating_user_field_name = "uploader"
+
+class AccountForm(ModelForm):
+    class Meta:
+        model = User
+        fields = ['username', 'first_name', 'email']
+        labels = {
+            'first_name': 'Real Name',
+        }
```

Comment:
From a user experience perspective it is weird that we store the person name in two places (here and in membership). As long as we do though, we may need to inform the user on the way it differs. As such I propose the following addition:

```suggestion
        }
        help_texts = {
            'first_name': 'This name is not connected to the name registered on your membership. Once membership is linked, the site will use the name registered on your membership instead.',
        }
```